src/api/v1/conversions.ts

```
@@ -0,0 +1,137 @@
+import db from "../../db/db";
+import { Jobs } from "../../db/types";
+import { handleConvert } from "../../converters/main";
+import { authMiddleware } from "../middleware/auth";
+import { outputDir, uploadsDir } from "../../index";
+import { ALLOW_UNAUTHENTICATED } from "../../helpers/env";
+
+export const conversions = new Elysia({ prefix: "/conversions" })
+  .use(authMiddleware)
+  .post(
+    "/",
+    async ({ body, set, ...ctx }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)